packages/hierarchy-builder/src/test/Utils.ts

```diff
@@ -96,3 +100,7 @@ export function createGetClassStub(schemas: IMetadataProvider) {
   };
   return { getClass: stub, stubClass };
 }
+
+export async function isMock(className: string): Promise<boolean> {
+  return className.includes("Parent");
+}
```

Comment:
I don't think this should be used at all:
- The name is way too obscure
- The behavior kind of depends on how the classes are set up in each test

Reply:
Removed